### PR TITLE
Pre dev

### PR DESCRIPTION
**Pull request by TheLuisAbarca**
## Pre-Dev

> In this branch, I have made the following changes:

**Day 1**

-     Select the API you are going to use.
-     Build files structure for your React app.
-     Prepare routes and navigation in your app.

**Day 2**

-     Make sure that a user can display a list of items and filter them. Data should come from the API.

**Day 3**

-     Style your components to match the design provided.

**Day 4**

-     Create the tests for the application.
-     Deploy the project and test for final details.
-     Record a video for your project.
-     Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,75 @@
...
+  <a href="https://github.com/TheLuisAbarca/pokestats-react_redux">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/TheLuisAbarca/pokestats-react_redux/development?color=blue&style=plastic">
+  </a>
+  <a href="https://github.com/TheLuisAbarca/pokestats-react_redux">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/TheLuisAbarca/pokestats-react_redux?color=orange&label=%E2%98%85%20stars%20&style=plastic">
+  </a>
+  <a href="https://github.com/TheLuisAbarca">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/TheLuisAbarca?color=yellow&logo=github&style=plastic">
+  </a>
+</p>
```
Comment (rloterh) [inline README.md:18]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [x] _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [x] The _code_ aspects in each procedure should be quoted with **backticks**.
- [x] Include necessary screenshots, project demo links where applicable so as to give a birds-eye view of your project
- [x] All _typos_ **should be corrected**. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)

File: README.md [hunk 2]
```diff
@@ -1,70 +1,75 @@
...
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Author
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+👤 **Luis Abarca**
 
-### `npm run eject`
+- GitHub: [@TheLuisAbarca](https://github.com/TheLuisAbarca)
```
Comment (rloterh) [inline README.md:90]: - [x] It's a nice and smart thing to give appropriate credit to **Nelson Sakwa**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <>
+      <div className="container-fluid min-width">
+        <audio /* eslint-disable-line */
```
Comment (rloterh) [inline src/App.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Luís, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🚦  

## Good Points 👍  
- Descriptive Pull Request
- Redux state management properly implemented
- Redux filter is well implemented
- Promise features properly implemented
- App is running as intended
- Nice UI design
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 3 above)
Comment (TheLuisAbarca) [reply src/App.js]: Thanks for your kindly and useful feedback @rloterh. :+1: :1st_place_medal:  I implement the solution for this in the commit a3ca22e

File: README.md (hunk 2 above)
Comment (TheLuisAbarca) [reply README.md:90]: Thanks for your kindly and useful feedback @rloterh. +1 1st_place_medal I implement the solution for this in the commit 8db61bc.

File: README.md (hunk 1 above)
Comment (TheLuisAbarca) [reply README.md:18]: Thanks for your kindly and useful feedback @rloterh. +1 1st_place_medal I implement the solution for this in the commit 8db61bc.

**Comment by omar-labana** [comment]
Hi @TheLuisAbarca, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**